src/neuroconv/nwbconverter.py

```diff
@@ -101,13 +101,20 @@ def get_metadata(self) -> DeepDict:
             metadata = dict_deep_update(metadata, interface_metadata)
         return metadata
 
-    def validate_metadata(self, metadata: Dict[str, dict]):
+    def validate_metadata(self, metadata: Dict[str, dict], append_mode: bool = False):
         """Validate metadata against Converter metadata_schema."""
         encoder = NWBMetaDataEncoder()
         # The encoder produces a serialized object, so we deserialized it for comparison
         serialized_metadata = encoder.encode(metadata)
         decoded_metadata = json.loads(serialized_metadata)
-        validate(instance=decoded_metadata, schema=self.get_metadata_schema())
+
+        metadata_schema = self.get_metadata_schema()
+        if append_mode:
+            # Eliminate required from NWBFile
+            nwbfile_schema = metadata_schema["properties"]["NWBFile"]
+            nwbfile_schema.pop("required", None)
```

Comment:
However, in truth, the entirety of `NWBFile` level metadata will be ignored if specified in this case (also `Subject`)

So any reason not to pop the entirety of the subschema for those?

Reply:
By ignore you mean that it won't be used? Those things are only used when creating the NWBFile, right? 

I think it is OK that if those values are passed in the metadata the check is performed. I see no lost from it save some small performance thing. I also feel that if I drop that version of the schema I might have to do the same operation in the metadata which seems dirtier.